packages/caip/src/constants.ts

```diff
@@ -177,3 +177,20 @@ export const VALID_ASSET_NAMESPACE: ValidAssetNamespace = Object.freeze({
     ASSET_NAMESPACE.slip44,
   ],
 })
+
+// We should prob change this once we add more chains
```

Comment:
lmeow

Reply:
one day i will get to refactor based on the vitriol from comment inline